Binary files caelum-0.6.1.orig/lib/libCaelum.so and caelum-0.6.1/lib/libCaelum.so differ
diff -NurEbBH -x bin -x CMakeCache.txt -x CMakeCache.txt.orig -x CMakeOutput.log -x CMakeCXXCompiler.cmake -x configurations.xml -x Package-Default.bash -x '*.includes' -x build.make -x flags.make -x link.txt -x '*.o' -x nbproject -x Progress -x cmake_install.cmake -x rorconfig.dir -x RoR.dir -x CMakeCache.txtr -x '*.cmake' -x '*.includecache' caelum-0.6.1.orig/main/src/CaelumPlugin.cpp caelum-0.6.1/main/src/CaelumPlugin.cpp
--- caelum-0.6.1.orig/main/src/CaelumPlugin.cpp	2011-08-22 16:39:29.000000000 +0400
+++ caelum-0.6.1/main/src/CaelumPlugin.cpp	2013-01-27 16:24:57.625469331 +0400
@@ -18,20 +18,22 @@
 along with Caelum. If not, see <http://www.gnu.org/licenses/>.
 */
 
+#include <OGRE/OgreSingleton.h>
+
 #include "CaelumPrecompiled.h"
 #include "CaelumPlugin.h"
 
-template<> Caelum::CaelumPlugin* Ogre::Singleton<Caelum::CaelumPlugin>::ms_Singleton = 0;
+template<> Caelum::CaelumPlugin* Ogre::Singleton<Caelum::CaelumPlugin>::msSingleton = 0;
 
 namespace Caelum
 {
 	CaelumPlugin* CaelumPlugin::getSingletonPtr () {
-        return ms_Singleton;
+        return msSingleton;
     }
 
     CaelumPlugin& CaelumPlugin::getSingleton () {  
-        assert (ms_Singleton);
-        return *ms_Singleton;  
+        assert (msSingleton);
+        return *msSingleton;  
     }
 
     extern "C" void CAELUM_EXPORT dllStartPlugin () {
